### PR TITLE
pylint

### PR DESCRIPTION
**Pull request by karstendick**

### CODE REVIEW

File: tap_gitlab/utils.py [hunk 1]
```diff
@@ -29,7 +30,7 @@ def wrapper(*args, **kwargs):
                 timer = threading.Timer(every, semaphore.release)
                 timer.setDaemon(True)  # allows the timer to be canceled on exit
                 timer.start()
-                return result
+            return result
```
Comment (karstendick) [inline tap_gitlab/utils.py]: I'm pretty sure this maintains the intent of this decorator while allowing exceptions to be passed along, but I could use another set of eyes on this.

Comment (mdelaurentis) [inline tap_gitlab/utils.py]: We should replace this with the ratelimit function in singer-python. See https://github.com/singer-io/singer-python/blob/master/singer/utils.py#L20

File: tap_gitlab/__init__.py [hunk 2]
```diff
@@ -51,6 +51,7 @@
     },
 }
 
+#pylint: disable=invalid-name
```
Comment (mdelaurentis) [inline tap_gitlab/__init__.py]: I would actually recommend changing these globals to all caps. That seems like a reasonable standard. @iterati and I have made that change in a couple other taps.

File: tap_gitlab/utils.py (hunk 1 above)
Comment (mdelaurentis) [inline tap_gitlab/utils.py]: Actually, can you please rip out this utils module and replace any references to it with references to the `singer` lib?

**Pull request merged**